### PR TITLE
chore(docs): add troubleshooting guide for mac arm64

### PR DESCRIPTION
**Pull request by AllenFang**
self explanatory, related to https://github.com/Kong/lua-resty-aws/issues/91

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,14 @@ To install from the git repo:
...
     make install
 
+### Troubleshooting
+If you are MacOS(arm64) user and encounter `fatal error: 'expat_config.h' file not found
+#include "expat_config.h"` error when install, please make sure you have `expat` in your system.   
+If this error still happen, please add `EXPAT_DIR` parameter when install:   
+```
+luarocks install lua-resty-aws EXPAT_DIR=/path/to/expat
+```
+
```
Comment (Tieske) [inline README.md:102]:
```suggestion
### Troubleshooting

MacOS has a known issue that the libexpat header file 'expat_config.h' is missing. If you run into that issue, install libexpat manually (eg. `brew install libexpat`). And then include the libexpat location when installing;
        luarocks install lua-resty-aws EXPAT_DIR=/path/to/expat

Details: https://github.com/lunarmodules/luaexpat/issues/32
```

Comment (Tieske) [reply README.md:102]: change of wording

Comment (AllenFang) [reply README.md]: done. thanks @Tieske

**Comment by windmgc** [comment]
@AllenFang Could you please sign the CLA in the PR comment?

**Review by Tieske — APPROVED** [review]

**Pull request merged**